src/__test__/app.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {
+  waitFor, render, fireEvent, screen, cleanup,
+} from './util';
+import App from '../App';
+import '@testing-library/jest-dom';
+
+afterEach(() => cleanup());
+
+describe('homepage', () => {
+  test('homepage renders', async () => {
+    const { container } = render(<App />);
+
+    await waitFor(() => screen.getByText('Afghanistan'));
+
+    expect(container).toMatchSnapshot();
+  });
+
+  test('API data renders', async () => {
+    render(<App />);
+
+    await waitFor(() => screen.getByText('Afghanistan'));
+
+    expect(screen.getByText('Algeria')).toBeInTheDocument();
+  });
+});
+
+describe('Details page', () => {
+  test('Details page should render', async () => {
+    const { container } = render(<App />);
+
+    await waitFor(() => screen.getByText('Algeria'));
+
+    fireEvent.click(screen.getByText('Algeria'));
+
+    expect(container).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] For some reason, I got an error when I tried to run the tests. It will be good if you could investigate the reason behind the failure of the test.

See screenshot :
![a1](https://user-images.githubusercontent.com/39523312/157723416-816c1925-f47e-420d-96fe-1292cdb3f6c3.png)

Reply:
Hi, this is for the snapshots, there are outdated because the data is updated, pressing U should solve it, otherwise I would have to not test snapshots

Reply:
Thanks, I solved it, I was using old snapshots, sorry